.github/workflows/policy-gate.yml

```
@@ -3,17 +3,14 @@ name: Policy Gate
 on:
   pull_request:
     types: [opened, synchronize, reopened]
-
-permissions:
-  contents: read
   pull-requests: read
```

actionlint (1.7.12)

[error] unknown Webhook event "pull-requests". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)